### PR TITLE
Fix and update links

### PR DESCRIPTION
**Pull request by marcuswin**

### CODE REVIEW

**Comment by rchen8** [comment]
Amazing thank you!

File: readme.md [hunk 1]
```diff
@@ -200,9 +183,8 @@
 ## Ethereum
 - https://ethstats.net
 - https://ethstats.io
-- https://amberdata.io
```
Comment (emlazzarin) [inline readme.md]: I'm going to add the more specific Amberdata dashboards in a PR.

File: readme.md [hunk 2]
```diff
@@ -213,8 +195,7 @@
 ## Exchanges
 - https://www.coingecko.com/en/exchanges
 - https://www.coingecko.com/en/derivatives
-- https://skew.com
```
Comment (emlazzarin) [inline readme.md]: Same — they have some public dashboards.

**Comment by emlazzarin** [comment]
I checked these and they all check out — really helpful, thanks!

**Pull request merged**